source/jormungandr/jormungandr/authentication.py

```diff
@@ -164,7 +184,14 @@ def cache_get_user(token):
     """
     if not can_connect_to_database():
         return None
-    return User.get_from_token(token, datetime.datetime.now())
+    try:
+        user = User.get_from_token(token, datetime.datetime.now())
+    except Exception as e:
+        logging.getLogger(__name__).error('No access to table User (error: {})'.format(e))
+        g.can_connect_to_database = False
+        return None
```

Comment:
Can you return a tuple (User, DbState)? So later, you don't have to test db's accessibility again.. https://github.com/CanalTP/navitia/pull/3652/files#diff-a7edc0c6cea6718d3c64b2d8a71e2848ec2ef57653932be2bbbee3e4179d8464R148

Reply:
As discussed, it will be very difficult to reuse db state for the following access to database.

Reply:
As discussed, it will be difficult to reuse the db state.